examples/src/content/jcr_root/content/core-components-examples/library/breadcrumb/breadcrumb/breadcrumb/.content.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would have probably created a structure like `breadcrumb/level1/level2` so it's easier to grasp.